conans/test/integration/command_v2/test_build_output.py

```diff
@@ -0,0 +1,21 @@
+import json
+import textwrap
+
+from conans.test.assets.genconanfile import GenConanfile
+from conans.test.utils.tools import TestClient
+
+
+def test_build_output_json():
```

Comment:
I'd probably put the test in ``integration/command/test_build.py``, so it is together with the other similar ``conan build`` command test